DEVELOPMENT.md

```diff
@@ -6,18 +6,36 @@ Install golang
 
 Then:
 
-    go get -d  ./...
+    go get -d ./...
     go generate
-    go install  ./...
 
-## Starting the Server
+## Run from code
 
 Start the server:
 
+    go run ./...
+
+## Build a binary
+
+Install the binary to `$GOPATH/bin/validator`:
+
+    go install ./...
+
+...then run it:
+
     validator
 
```

Comment:
I didn't test these steps, but the rest worked.

Reply:
I did, so I guess that should be good enough 🙂